reflekt-plugin/src/main/kotlin/io/reflekt/plugin/generation/bytecode/Util.kt

```diff
@@ -0,0 +1,53 @@
+package io.reflekt.plugin.generation.bytecode
+
+import org.jetbrains.org.objectweb.asm.Label
+import org.jetbrains.org.objectweb.asm.Type
+import org.jetbrains.org.objectweb.asm.commons.InstructionAdapter
+
+/**
+ * Push array of non-primitive type on stack.
+ */
+fun <T> InstructionAdapter.pushArray(arrayType: Type, items: List<T>, pushItem: InstructionAdapter.(T) -> Unit, checkForNull: Boolean = true) {
+    iconst(items.size)
+    newarray(arrayType)
+    items.forEachIndexed { index, item ->
+        dup()
+        iconst(index)
+        pushItem(item)
+        checkCast(arrayType, checkForNull)
+        astore(InstructionAdapter.OBJECT_TYPE)
+    }
+}
+
+fun InstructionAdapter.checkCast(castType: Type, checkForNull: Boolean) {
+    if (checkForNull) {
+        val nonNullLabel = Label()
+
+        dup()
+        ifnonnull(nonNullLabel)
+        anew(Type.getObjectType("kotlin/TypeCastException"))
```

Comment:
You use different consts like `kotlin/TypeCastException` for paths. I think we can create storage for them or something like this. I mean, if you use string constant you can make a mistake. But if we use variables, we are sure the same variable has the same value in the different places